src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -581,7 +581,7 @@ public static void renderTileItem(Item item, int metadata)
     }
 
     /**
-     * @deprecated Will be removed as soon as possible, hopefully 1.9.
+     * @deprecated Will be removed as soon as possible, hopefully 1.9.  See {@link Item#getTileEntityItemStackRenderer()}.
```

Comment:
since we're now 3 major versions past 1.9, maybe we can remove that part of the comment lol